### PR TITLE
Support to get DocDB Instance details

### PR DESCRIPTION
**Pull request by nvnyale**

### CODE REVIEW

File: api/orchestration.go [hunk 1]
```diff
@@ -186,9 +186,15 @@ func (o *docDBOrchestrator) documentDBDetails(ctx context.Context, name string)
 		return nil, apierror.New(apierror.ErrNotFound, "cluster not found in our org", nil)
 	}
 
+	instances, err := o.docdbClient.GetDocDBInstances(ctx, name)
```
Comment (tenyo) [inline api/orchestration.go:189]: I'm not sure why we didn't have this in the response originally. Did you need it for something specifically? Can you add the output of `GET /v1/docdb/{account}/{name}` (with the Instances) to the README, so I can see what it look like?

File: README.md [hunk 2]
```diff
@@ -209,50 +209,130 @@ GET `/v1/docdb/{account}/{name}`
...
         "EnabledCloudwatchLogsExports": null,
-        "Endpoint": "mydocdb.cluster-z0ukc6s0rmbg.us-east-1.docdb.amazonaws.com",
+        "Endpoint": "mydocdbtest6.cluster-c9ukc6s0rmbg.us-east-1.docdb.amazonaws.com",
         "Engine": "docdb",
         "EngineVersion": "4.0.0",
-        "HostedZoneId": "ZZXXYY5TT8WVW",
-        "KmsKeyId": "arn:aws:kms:us-east-1:123456789012:key/11aa0000-8fcb-4e65-abb4-eed7f4a012f7",
-        "LatestRestorableTime": "2021-08-05T13:23:43.551Z",
+        "HostedZoneId": "ZNKXH85TT8WVW",
+        "KmsKeyId": "arn:aws:kms:us-east-1:846761448161:key/91c73e09-8fcb-4e65-abb4-eed7f4a012f7",
```
Comment (tenyo) [inline README.md]: 👀 use `123456789012` for the account

File: README.md [hunk 3]
```diff
@@ -209,50 +209,130 @@ GET `/v1/docdb/{account}/{name}`
...
     },
+    "Instances": [
+        {
+            "AutoMinorVersionUpgrade": true,
+            "AvailabilityZone": "us-east-1a",
+            "BackupRetentionPeriod": 2,
+            "CACertificateIdentifier": "rds-ca-2019",
+            "CopyTagsToSnapshot": false,
+            "DBClusterIdentifier": "mydocdbtest6",
+            "DBInstanceArn": "arn:aws:rds:us-east-1:846761448161:db:mydocdbtest6-1",
```
Comment (tenyo) [inline README.md]: 👀

File: README.md [hunk 4]
```diff
@@ -209,50 +209,130 @@ GET `/v1/docdb/{account}/{name}`
...
+            "EnabledCloudwatchLogsExports": null,
+            "Endpoint": {
+                "Address": "mydocdbtest6-1.c9ukc6s0rmbg.us-east-1.docdb.amazonaws.com",
+                "HostedZoneId": "ZNKXH85TT8WVW",
+                "Port": 27017
+            },
+            "Engine": "docdb",
+            "EngineVersion": "4.0.0",
+            "InstanceCreateTime": "2022-07-26T18:32:35.634Z",
+            "KmsKeyId": "arn:aws:kms:us-east-1:846761448161:key/91c73e09-8fcb-4e65-abb4-eed7f4a012f7",
```
Comment (tenyo) [inline README.md]: 👀

File: README.md [hunk 5]
```diff
@@ -209,50 +209,130 @@ GET `/v1/docdb/{account}/{name}`
...
             "us-east-1a"
         ],
-        "BackupRetentionPeriod": 1,
-        "ClusterCreateTime": "2021-08-05T13:23:03.003Z",
-        "DBClusterArn": "arn:aws:rds:us-east-1:123456789012:cluster:mydocdb",
-        "DBClusterIdentifier": "mydocdb",
+        "BackupRetentionPeriod": 2,
+        "CloneGroupId": null,
+        "ClusterCreateTime": "2022-07-26T18:27:59.637Z",
+        "DBClusterArn": "arn:aws:rds:us-east-1:846761448161:cluster:mydocdbtest6",
```
Comment (tenyo) [inline README.md]: 👀

**Review by tenyo — COMMENTED** [review]
Let's edit the account number, otherwise it's good 👍

File: README.md (hunk 3 above)
Comment (nvnyale) [reply README.md]: done.

**Pull request merged**